atlp-rwanda/c4-barefoot-frontend
# Ft individual travel history

# Description

This set up a project from scratch without using npx create-react-app


## Type of change

Please check the options that are relevant.


# How should this be manually tested or reproduced?

- Clone this repo
- Run the command `npm i`
- Run the command `npm start`
- You need to login as a requester or manager, once you are logged in you go to travel-history then it displays all the trips made, if you want to see the booked accommodation you click on details it will direct you to the page that view the all infos. when you login and no trips made it will give you a message no trips available.

# Checklist:


# Screenshots (if applicable)
